I'm unable to get the plugin to complain on my module under test

I currently have `pylint` + `py.test` setup in my project. The basic structure is...

```
project-dir/
  setup.cfg
  .pylintrc
  mypackage/
  tests/
    mypackage/
```

I run py.test using a command like so

```
PYTHONPATH=. py.test --pylint --pylint-rcfile=.pylintrc
```

I don't get an error from `py.test` when I know that files in `mypackage` are not clean.

I'm using versions
py.test==2.8.2
pytest-pylint==0.4.0
pylint==1.1.0
atroid==1.0.1
Python=2.7.6
OS Ubuntu 14.04 LTS

That is strange.  Do you have the source anywhere I can take a look at, or at least the pylintrc file?

I just generated a simple test project with the same structure as yours, and everything seemed to check out fine:
```shell
PYTHONPATH=. py.test --pylint --pylint-rcfile=.pylintrc
======================================= test session starts =======================================
platform darwin -- Python 2.7.10, pytest-2.8.2, py-1.4.30, pluggy-0.3.1
rootdir: /Users/carson/tmp/test, inifile:
plugins: pylint-0.4.0
collected 6 items

mypackage/__init__.py .
mypackage/blah.py F
test/__init__.py .
test/test_blah.py F
test/mypackage/__init__.py .
test/mypackage/blah.py F

============================================ FAILURES =============================================
___________________________________ [pylint] mypackage/blah.py ____________________________________
C:  1, 0: Unnecessary parens after 'print' keyword (superfluous-parens)
C:  1, 0: Missing module docstring (missing-docstring)
-------------------------------------- Captured stderr call ---------------------------------------
Warning: option include-ids is deprecated and ignored.
Warning: option symbols is deprecated and ignored.
___________________________________ [pylint] test/test_blah.py ____________________________________
C:  1, 0: Unnecessary parens after 'print' keyword (superfluous-parens)
C:  1, 0: Missing module docstring (missing-docstring)
-------------------------------------- Captured stderr call ---------------------------------------
Warning: option include-ids is deprecated and ignored.
Warning: option symbols is deprecated and ignored.
_________________________________ [pylint] test/mypackage/blah.py _________________________________
C:  1, 0: Unnecessary parens after 'print' keyword (superfluous-parens)
C:  1, 0: Missing module docstring (missing-docstring)
-------------------------------------- Captured stderr call ---------------------------------------
Warning: option include-ids is deprecated and ignored.
Warning: option symbols is deprecated and ignored.
=============================== 3 failed, 3 passed in 0.16 seconds ================================
```

where each of the failing files are just one print statement.

Hey, so I did some snooping around the configuration files. I've narrowed it down to my setup.cfg. It looks like...

```
[pytest]
addopts = -rsxX -q
testpaths = tests
minversion = 2.8
```

It seems if I remove the `testpaths = tests`, then pylint plugin for py.test correctly runs it against `mypackage`. With it, it skips the `mypackage` directory.

btw, I started getting a new error with this work around. Specifically for each file that pylint runs on, I get this message

```
/usr/local/lib/python2.7/dist-packages/pytest_pylint.py:122: in runtest
    lint.Run(args_list, reporter=reporter, exit=False)
/usr/lib/python2.7/dist-packages/pylint/lint.py:1040: in __init__
    linter.check(args)
/usr/lib/python2.7/dist-packages/pylint/lint.py:626: in check
    self.check_astroid_module(astroid, walker, rawcheckers, tokencheckers)
/usr/lib/python2.7/dist-packages/pylint/lint.py:712: in check_astroid_module
    walker.walk(astroid)
/usr/lib/python2.7/dist-packages/pylint/utils.py:694: in walk
    cb(astroid)
/usr/lib/python2.7/dist-packages/pylint/checkers/base.py:905: in visit_module
    self._check_docstring('module', node)
/usr/lib/python2.7/dist-packages/pylint/checkers/base.py:941: in _check_docstring
    self.add_message('missing-docstring', node=node, args=(node_type,))
/usr/lib/python2.7/dist-packages/pylint/checkers/__init__.py:100: in add_message
    self.linter.add_message(msg_id, line, node, args)
/usr/lib/python2.7/dist-packages/pylint/utils.py:445: in add_message
    self.reporter.add_message(msgid, (path, module, obj, line or 1, col_offset or 0), msg)
/usr/local/lib/python2.7/dist-packages/pytest_pylint.py:31: in add_message
    raise NotImplementedError
E   NotImplementedError
```

--- a/pytest_pylint.py
+++ b/pytest_pylint.py
@@ -2,17 +2,17 @@
 from __future__ import unicode_literals
 from __future__ import absolute_import
 from os.path import exists
+from six.moves.configparser import (  # pylint: disable=import-error
+    ConfigParser,
+    NoSectionError,
+    NoOptionError
+)
 
 from pylint import lint
 from pylint.config import PYLINTRC
 from pylint.interfaces import IReporter
 from pylint.reporters import BaseReporter
 import pytest
-from six.moves.configparser import (  # pylint: disable=import-error
-    ConfigParser,
-    NoSectionError,
-    NoOptionError
-)
 
 
 class ProgrammaticReporter(BaseReporter):
@@ -64,7 +64,7 @@ def pytest_collect_file(path, parent):
     config = parent.config
     if not config.option.pylint:
         return
-    if not path.ext == ".py":
+    if path.ext != ".py":
         return
     # Find pylintrc to check ignore list
     pylintrc_file = config.option.pylint_rcfile or PYLINTRC

--- a/setup.py
+++ b/setup.py
@@ -19,7 +19,7 @@ setup(
     url='https://github.com/carsongee/pytest-pylint',
     py_modules=['pytest_pylint'],
     entry_points={'pytest11': ['pylint = pytest_pylint']},
-    install_requires=['pytest>=2.4', 'pylint', 'six'],
+    install_requires=['pytest>=2.4', 'pylint>=1.4.5', 'six'],
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'Intended Audience :: Developers',

